plugins/mcreator-core/datalists/sounds.yaml

```diff
@@ -996,3 +997,188 @@
+- item.honeycomb.wax_on
+- entity.axolotl.attack
+- entity.axolotl.death
+- entity.axolotl.hurt
+- entity.axolotl.idle_air
+- entity.axolotl.idle_water
+- entity.axolotl.splash
+- entity.axolotl.swim
+- block.azalea.break
+- block.azalea.fall
+- block.azalea.hit
+- block.azalea.place
+- block.azalea.step
+- block.azalea_leaves.break
+- block.azalea_leaves.fall
+- block.azalea_leaves.hit
+- block.azalea_leaves.place
+- block.azalea_leaves.step
+- block.big_dripleaf.break
+- block.big_dripleaf.fall
+- block.big_dripleaf.hit
+- block.big_dripleaf.place
+- block.big_dripleaf.step
+- item.bucket.empty_powder_snow
+- item.bucket.fill_axolot
+- item.bucket.fill_powder_snow
+- block.cake.add_candle
+- block.calcite.break
+- block.calcite.fall
+- block.calcite.hit
+- block.calcite.place
+- block.calcite.step
+- block.candle.break
+- block.candle.extinguish
+- block.candle.fall
+- block.candle.hit
+- block.candle.place
+- block.candle.step
+- block.cave_vines.break
+- block.cave_vines.fall
+- block.cave_vines.hit
+- block.cave_vines.pick_berries
+- block.cave_vines.place
+- block.cave_vines.step
+- block.copper.break
+- block.copper.fall
+- block.copper.hit
+- block.copper.place
+- block.copper.step
+- block.deepslate_bricks.break
+- block.deepslate_bricks.fall
+- block.deepslate_bricks.hit
+- block.deepslate_bricks.place
+- block.deepslate_bricks.step
+- block.deepslate.break
+- block.deepslate.fall
+- block.deepslate.hit
+- block.deepslate.place
+- block.deepslate.step
+- block.deepslate_tiles.break
+- block.deepslate_tiles.fall
+- block.deepslate_tiles.hit
+- block.deepslate_tiles.place
+- block.deepslate_tiles.step
+- block.dripstone_block.break
+- block.dripstone_block.fall
+- block.dripstone_block.hit
+- block.dripstone_block.place
+- block.dripstone_block.step
+- block.pointed_dripstone.break
+- block.pointed_dripstone.fall
+- block.pointed_dripstone.hit
+- block.pointed_dripstone.place
+- block.pointed_dripstone.step
+- block.pointed_dripstone.land
+- block.pointed_dripstone.drip_lava
+- block.pointed_dripstone.drip_water
+- block.pointed_dripstone.drip_lava_into_cauldron
+- block.pointed_dripstone.drip_water_into_cauldron
+- block.big_dripleaf.tilt_down
+- block.big_dripleaf.tilt_up
+- block.flowering_azalea.break
+- block.flowering_azalea.fall
+- block.flowering_azalea.hit
+- block.flowering_azalea.place
+- block.flowering_azalea.step
+- block.roots.break
+- block.roots.fall
+- block.roots.hit
+- block.roots.place
+- block.roots.step
+- item.glow_ink_sac.use
+- entity.glow_item_frame.add_item
+- entity.glow_item_frame.break
+- entity.glow_item_frame.place
+- entity.glow_item_frame.remove_item
+- entity.glow_item_frame.place_item
+- entity.glow_squid.ambient
+- entity.glow_squid.death
+- entity.glow_squid.hurt
+- entity.glow_squid.squirt
+- entity.goat.ambient
+- entity.goat.death
+- entity.goat.eat
+- entity.goat.hurt
+- entity.goat.long_jump
+- entity.goat.milk
+- entity.goat.prepare_ram
+- entity.goat.ram_impact
+- entity.screaming.ambient

```

Comment:
I would change:

- entity.screaming.ambient
- entity.screaming.death
- entity.screaming.eat
- entity.screaming.hurt
- entity.screaming.long_jump
- entity.screaming.milk
- entity.screaming.prepare_ram
- entity.screaming.ram_impact


to something like:

- entity.goat.screaming.ambient
- entity.goat.screaming.death
- entity.goat.screaming.eat
- entity.goat.screaming.hurt
- entity.goat.screaming.long_jump
- entity.goat.screaming.milk
- entity.goat.screaming.prepare_ram
- entity.goat.screaming.ram_impact

since this is closer to Minecraft's mappings and also gives more info about the sound.